src/lib/fs-utils.ts

```diff
@@ -82,3 +83,24 @@ export async function unlinkAll(...paths: string[]): Promise<void> {
 export function getPathOnHost(...paths: string[]): string[] {
 	return paths.map((p: string) => path.join(constants.rootMountPoint, p));
 }
+
+/**
+ * Change modification and access time of the given file.
+ * It creates an empty file if it does not exist
+ */
+export const touch = (file: string, time = new Date()) =>
+	fs.utimes(file, time, time).catch((e) =>
```

Comment:
Are there supposed to be 2 `time` params here, or should they be `atime` and `mtime` instead for clarity?

Reply:
You mean as input for the touch function? Not really, this is mimicking the touch command on Unix that modifies both mtime and atime to the current time. I added the time as an arg since I think it might be useful in the future

Reply:
Oh wait, I think I misunderstood, but part of my answer still applies. We want to pass the same value for the `atime` and `mtime` args to the `fs.utimes` function. We could create two variables with the same value for clarity but that might be a little redundant.